tests/main/snap-userd-desktop-app-autostart/task.yaml

```diff
@@ -29,12 +30,10 @@ execute: |
     test -e ~/snap/test-snapd-xdg-autostart/current/foo-autostarted
 
     if [[ "$SPREAD_SYSTEM" != ubuntu-14.04-* ]]; then
-        journalctl --flush
-        test "$(journalctl -t foo.desktop --after-cursor=$cursor | wc -l)" -eq 1
+        test "$(journalctl -t foo.desktop --cursor "$cursor" | wc -l)" -gt 0
```

Comment:
You have a function for this, no? Perhaps the function is too trivial and should be removed.

Reply:
The function that we already have checks against the predefined cursor, in this case we are checking agains another cursor. I'll update the function to reuse it.

Reply:
I think we had a function that took the cursor as argument but perhaps I was mistaken,